src/app.py

```diff
@@ -170,6 +170,26 @@ def get_entity(identifier):
             msg += str(x)
         abort(400, msg)
 
+
+# Get data_access_level for a given entity uuid (Donor/Sample/Dataset)
+@app.route('/entity-access-level/<uuid>', methods = ['GET'])
+@secured(groups="HuBMAP-read")
+def get_entity_access_level(uuid):
+    try:
+        dataset = Dataset(app.config)
+        return dataset.get_entity_access_level(uuid)
+    except HTTPException as hte:
+        msg = "HTTPException during get_entity_access_level HTTP code: " + str(hte.get_status_code()) + " " + hte.get_description() 
+        logger.warn(msg, exc_info=True)
+        print(msg)
+        return Response(hte.get_description(), hte.get_status_code())
+    except CypherError as ce:
+        msg = 'A Cypher error was encountered when calling dataset.get_entity_access_level(), check log file for detail'
+        logger.error(msg, exc_info=True)
+        print(msg)
+        return Response(msg, 500)
```

Comment:
This should catch a general Exception here instead of just a CypherError, that way if unexpected errors occur we'll cleanly handle